### ISSUE TITLE
Negative percentage values still flagged by shorthand-values

### ISSUE THREAD

**Issue by attila** [sasstools/sass-lint#435]
This could be related to https://github.com/sasstools/sass-lint/pull/429 but

```scss
.foo {
  margin: -10% -20%;
}
```

is flagged by "shorthand-values" in latest develop.

Command output:

```
$ ./$ node_modules/.bin/sass-lint -v test.scss

test.scss
  2:11  warning  Property `margin` should be written more concisely as `-` instead of `- -`  shorthand-values

✖ 1 problem (0 errors, 1 warning)
```

**Comment by DanPurdy** [comment]
Confirmed, looks like something to do with percentage values and their structure in gonzales. Will look into it, thanks @attila .

**Comment by DanPurdy** [comment]
sorted in #441 will be part of the 1.4.0 release scheduled for this week
